style.css

```diff
@@ -67,3 +63,184 @@ header {
   flex-direction: column;
   padding: 25px;
 }
+
+.myAbilities {
+  background-image: url(./assets/ellipse2.svg);
+  background-repeat: no-repeat;
+  background-position: right;
+}
+
+.mySkills {
+  margin-top: 25px;
+  margin-left: 60px;
+}
+
+.abilities {
+  padding-left: 0;
+  text-decoration: none;
+}
+
+.abilities li {
+  list-style: none;
+}
+
+li a {
+  text-decoration: none;
+  color: white;
+  font-family: 'Roboto', sans-serif;
+  font-weight: 100;
+  font-size: 14px;
+  line-height: 30px;
+}
+
+a h2 {
+  text-decoration: none;
+  color: #9490ff;
+  font-family: 'Roboto', sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+}
+
+.skillsSection {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.line {
+  width: 300px;
+  margin-left: 30px;
+}
+
+#github,
+#linkedIn,
+#angellist,
+#twitter,
+#medium {
+  width: 20px;
+  margin: 10px;
+}
+
+.profiles {
+  list-style: none;
+  display: flex;
+  padding-left: 0;
+  justify-content: center;
+  align-items: center;
+}
+
+.work {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+#downArrow {
+  height: 20px;
+  margin: 5px;
+}
+
+.work h2 {
+  font-family: 'Roboto', sans-serif;
+  font-weight: 500;
+  font-size: 20px;
+  line-height: 32px;
+  color: #c4c4c4;
+  margin: 5px;
+}
+
+.myRecentWorks {
+  font-family: 'Roboto', sans-serif;
+  font-weight: 500;
+  font-size: 40px;
+  color: #9490ff;
+  text-align: left;
+  margin: 0;
+  padding-top: 70px;
+  padding-left: 30px;
+  text-transform: uppercase;
+}
+
+.languagesUsed {
+  list-style: none;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  padding-left: 0;
+}
+
+.languagesUsed li {
+  color: white;
+  margin: 10px;
+  font-family: 'Inter', sans-serif;
+  font-weight: 500;
+  font-size: 15px;
+}
+
+.myProjects {
+  display: grid;
+}
+
+.multiPost {
+  font-family: 'Roboto', sans-serif;
+  font-weight: 500;
+  font-size: 32px;
+  color: white;
+}
+
+.projectBrief {
+  font-family: 'Inter', sans-serif;
+  font-weight: 400;
+  font-size: 20px;
+  text-align: left;
+  color: #dcdce8;
+}
+
+#project1 {
+  background-image: url(./assets/pro1image.svg);
+  background-repeat: no-repeat;
+  background-position: left;
+  padding: 20px;
+}
+
+#project2 {
+  background-image: url(./assets/pro2image.svg);
+  background-repeat: no-repeat;
+  background-position: right;
+  padding: 20px;
+}
+
+#project3 {
+  background-image: url(./assets/pro3image.svg);
+  background-repeat: no-repeat;
+  background-position: left;
+  padding: 20px;
+}
+
+#project4 {
+  background-image: url(./assets/pro4image.svg);
+  background-repeat: no-repeat;
+  background-position: right;
+  padding: 20px;
+}
```

Comment:
- [ ] Great job while implementing these projects container. They should align the items inside of it by using a flexbox. Can you please use the CSS flexbox to lay out the contents of these projects?
   - They need to have something like this:
   ```CSS
  #project {
     display: flex;
     ...
   }
   ```